KsApi/lib/Route.swift

```diff
@@ -121,7 +121,7 @@ internal enum Route {
       return (.GET, url, [:], nil)
 
     case .config:
-      return (.GET, "/v1/app/ios/config", [:], nil)
+      return (.GET, "/v1/app/ios/config", ["no_cache": "\(Date().timeIntervalSince1970)"], nil)
```

Comment:
I wonder if in the future we could update this to instead use `URLRequest`'s `cachePolicy` of `reloadIgnoringLocalAndRemoteCacheData` 🤔

Reply:
Yeah good point - this would require a bit more work because it would mean having a separate `URLSession` for just this request. I think this should be fine for now and hopefully we can remove this when we disable caching for this endpoint at the backend.